deconz/config.yaml

```
@@ -1,7 +1,9 @@
+---
 version: 6.12.0
 slug: deconz
 name: deCONZ
-description: Control a Zigbee network with ConBee or RaspBee by Dresden Elektronik
+description: >
```

```suggestion
description: >-
```
Use `>-` to strip whitespace for these (same goes for all others like it)